### PR TITLE
Schemas and unit tests for 22-10216

### PR DESCRIPTION
**Pull request by fatmakhan0395**
# New schema
_Please describe the new schema that is being added and include links to any relevant issues to help future developers understand the schema and related code._

_Please ensure you have incremented the version in_ `package.json`.

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/

## Pull Requests to update the schema in related repositories
_After you've merged your changes to vets-json-schema you'll need to make PR's to vets-website and vets-api. Please link them here._

- https://github.com/department-of-veterans-affairs/vets-api/pull/
- https://github.com/department-of-veterans-affairs/vets-website/pull/

### CODE REVIEW

File: src/schemas/22-10216/schema.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          $ref: '#/definitions/date',
+        },
+      },
+    },
+    studentRatioCalcChapter: {
+      type: 'object',
+      required: ['beneficiaryStudent', 'numOfStudent', 'dateOfCalculation'],
+      properties: {
+        beneficiaryStudent: {
+          type: 'number',
```
Comment (GovCIOLiz) [inline src/schemas/22-10216/schema.js:36]: integer

File: src/schemas/22-10216/schema.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        },
+        numOfStudent: {
+          type: 'number',
```
Comment (GovCIOLiz) [inline src/schemas/22-10216/schema.js:39]: integer

File: src/schemas/22-10216/schema.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        institutionName: {
+          type: 'string',
+        },
+        facilityCode: {
+          type: 'string',
+        },
+        termStartDate: {
+          $ref: '#/definitions/date',
+        },
+        dateOfCalculations: {
```
Comment (GovCIOLiz) [inline src/schemas/22-10216/schema.js:26]: ?

File: test/schemas/22-10216/schema.spec.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      'institutionName',
+      'facilityCode',
+      'termStartDate',
+    ]);
+    expect(schema.properties.studentRatioCalcChapter.required).to.deep.equal([
+      'beneficiaryStudent',
+      'numOfStudent',
+      'dateOfCalculation',
+    ]);
+  });
```
Comment (GovCIOLiz) [inline test/schemas/22-10216/schema.spec.js:21]: Add `schemaTestHelper.testValidandInvalid` to test valid/invalid data examples

File: src/schemas/22-10216/schema.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        beneficiaryStudent: {
+          type: 'number',
+        },
+        numOfStudent: {
+          type: 'number',
+        },
+        dateOfCalculation: {
+          $ref: '#/definitions/date',
+        },
+      },
```
Comment (GovCIOLiz) [inline src/schemas/22-10216/schema.js:44]: if we are passing the calculated percentage to the backend, add the key with type: number for a float value

**Review by ericboehs — APPROVED** [review]

**Pull request merged**